### ISSUE TITLE
Agent Dialog UI Update & Proper Branding

### ISSUE THREAD

**Issue by dsibiski** [AutoElevate/roadmap#11]

**Comment by dts-alex** [comment]
Would be good if this matched Company settings, so that it could be branded by company, especially in a co-managed IT setup.

**Comment by dsibiski** [comment]
@dts-alex Yes, it definitely will. With "Multi-Level Settings" pretty much everything will be able to be configured down to the Company/Location or even Computer level. (If you aren't on early access and want those enabled for you now, let me know)
